hack/make-rules/verify.sh

```diff
@@ -54,6 +54,13 @@ if [[ ${EXCLUDE_GODEP:-} =~ ^[yY]$ ]]; then
     )
 fi
 
+# Exclude readonly package check in certain cases, aka, in CI we don't care and a readonly package won't be touched
```

Comment:
I think we intend to exclude this only in periodic jobs? might make sense to say that explicitly.

Reply:
done